module/sidebar/components/menu-component.js

```diff
@@ -103,11 +103,51 @@ export default class MenuComponent extends Component {
             }
         })
 
-        this.element.children('#menu-content').children('.pokeball-item').on("click", (event) => {
+        this.element.children('#menu-content').children('.pokeball-item').on("click", async(event) => {
             const { entityId, ballName, ownerId } = event.target.dataset;
 
             const owner = game.actors.get(ownerId);
-            game.ptu.utils.throwPokeball(owner, game.user?.targets?.first(), owner?.items.get(entityId));
+
```

Comment:
This will ask the player if they can throw the pokeball. Not the GM. You require the run-as-gm API for that.

I don't have the time to currently explaint that so won't be merging it in for the final v10 beta but can tackle it next week